ncss_api/convert.py

```diff
@@ -42,19 +41,19 @@ def numerals_api():
               example: eleven
   """
   value = request.args.get('value')
-  to = request.args.get('to', 'cardinal')
+  to = request.args.get('to', 'words')
 
   if value:
-    if to == 'cardinal' or to == 'words':
+    if to == 'words':
```

Comment:
@kennib not strictly necessary, but the api should follow the docs as closely as possible to avoid https://www.hyrumslaw.com/